.prettierignore

```diff
@@ -0,0 +1,2 @@
+external
+.*
```

Comment:
Why do you need this?
It looks like `external` is actually now code written by us, so it should be prettified.

Reply:
Ok, then please revert changes from https://github.com/XRPLF/rippled/pull/5657 done to this folder
And I think it's better to be done in a separate PR (including changes to this file as well)

Reply:
Duh I totally missed that PR :-|

Reply:
@bthomee I think @mathbunnyru is correct, we should revert changes from that other commit under `external`